mod/cli/pkg/config/client.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BUSL-1.1
+//
```

Comment:
what was your plan? would a client folder make more sense?

Reply:
eh no pref i just moved this cause i hate utils.go as a concept and its like just a convenience func for cosmos client config